dev/travis-before_install-encryption.sh

```diff
@@ -0,0 +1,27 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I'm afraid that in order for this to work, we'll need https://github.com/apache/parquet-mr/commit/3d8eae78291ace9d9a12839a932c52a785b0b308 too. @ggershinsky should we rebase this branch onto master?

Reply:
sounds good to me. this won't affect anything. should I close the pr, and open a new one against the parquet-mr/master?

Reply:
@nandorKollar can I ask you to rebase this branch? Or let me know if you prefer me to create a pull request for that.

Reply:
@ggershinsky done, let me retrigger Travis to see the test results.

Reply:
Seems that Travis fails because the default JDK version is now 11, #665 fixes it.